### PR TITLE
Offer's CRUD structure

### PR DESCRIPTION
**Pull request by oramasangel**
# Description
 I've created the offer's crud system
 - Creating offer's views
 - Creating offer's controller
 - Creating offer's routes
 - Creating relationships on offers model
 - Creating relationships on products model

 Examples:
 - LOREM IPSUN
 
 # Testing
LOREM IPSUN
 
 Example:
LOREM IPSUN
  
  ## Screenshots

### CODE REVIEW

**Review by jairolan — APPROVED** [review]

File: local_sales/config/database.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ development:
   username: <%= ENV['DB_USERNAME'] %>
   password: <%= ENV['DB_PASSWORD'] %>
 
+
```
Comment (jclopezdev) [inline local_sales/config/database.yml:31]: I recommend you to remove these changes that don't add value to the PR, or you could, you could also think about adding it to the `.gitignore` file
https://stackoverflow.com/a/33484222

File: local_sales/.env.example [hunk 2]
```diff
@@ -1,4 +0,0 @@
-DB_NAME=xxxxxx
```
Comment (jclopezdev) [inline local_sales/.env.example:1]: Why are you removing this file? 
If you need to remove it you can do it in another PR, to maintain only the `offers crud` changes in this PR

File: local_sales/config/database.yml (hunk 1 above)
Comment (oramasangel) [reply local_sales/config/database.yml:31]: we use a .ENV to set the settings for our postgre's databse, but i forgot that we used that file (.ENV), and i changed the database.yml, but now i changed the .ENV and removed the database.yml from the .gitignore, cause we have the .ENV in that file

Comment (juliocabrera820) [reply local_sales/config/database.yml:31]: @jclopezdev we are using dotenv-rails gem

File: local_sales/app/views/offers/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<p id="notice"><%= notice %></p>
```
Comment (jclopezdev) [inline local_sales/app/views/offers/index.html.erb:1]: What do you think of moving this line to the [`application.html.erb`](https://github.com/BC-MAY-21-ROR/proyecto-proy-t02/blob/master/local_sales/app/views/layouts/application.html.erb) file to avoid duplicating it?

**Review by YilKanda — APPROVED** [review]
Just to highlight: try to keep your PR as simple as possible, if you are indicating that the PR is from the CRUD of offers, that only the respective changes to what is indicated appear. If you made a change to the database configuration, that would correspond to another PR. The purpose that in the future you identify in which part of your code there is a change that you are looking for.

**Review by JoseRicardoJimenez — DISMISSED** [review]
LG (Y)

**Review by juliocabrera820 — DISMISSED** [review]

**Review by juliocabrera820 — APPROVED** [review]

**Review by JoseRicardoJimenez — APPROVED** [review]

**Review by jairolan — APPROVED** [review]
